src/states/elder/mod.rs

```diff
@@ -1765,22 +1519,15 @@ impl Approved for Elder {
         online_payload: OnlinePayload,
         outbox: &mut dyn EventBox,
     ) -> Result<(), RoutingError> {
-        if self.chain.try_approve_candidate(&online_payload) {
-            let OnlinePayload {
-                new_public_id,
-                client_auth,
-                ..
-            } = online_payload;
+        //if self.chain.try_approve_candidate(&online_payload) {
```

Comment:
Do we need to keep this commented out code?

Reply:
I'll see - I probably left it there initially, because I wasn't sure if it should go, and it became completely useless later.